### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wty4427300**
Update document links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,8 +25,8 @@ TiDB-Binlog is a tool used to collect [TiDB's](https://github.com/pingcap/tidb)
     Replay from any history point.
 
 ## Documentation
-+ [Detailed documentation](https://pingcap.com/docs/tools/tidb-binlog-cluster/)
-+ [简体中文](https://pingcap.com/docs-cn/tools/tidb-binlog-cluster/)
++ [Detailed documentation](https://pingcap.com/docs/dev/reference/tidb-binlog-overview/)
++ [简体中文](https://pingcap.com/docs-cn/v3.0/reference/tools/tidb-binlog/overview/#tidb-binlog-%E7%AE%80%E4%BB%8B)
```
Comment (july2993) [inline README.md]:
```suggestion
+ [简体中文](https://pingcap.com/docs-cn/dev/reference/tools/tidb-binlog/overview/)
```

**Comment by GregoryIan** [comment]
@wty4427300 , thanks for your PR, please address the comment

**Comment by leoppro** [comment]
ping @wty4427300

**Comment by GregoryIan** [comment]
thanks @wty4427300 again. This PR has not been updated for too long, I helped it,  please don't mind.

**Pull request merged**